### PR TITLE
fix issue 16241 - std.xml mistakenly disallows "==" in comments but a…

### PR DESCRIPTION
**Pull request by aG0aep6G**
…llows "--"

### CODE REVIEW

**Comment by burner** [comment]
LGTM

**Comment by burner** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
Auto-merge toggled off

File: std/xml.d [hunk 1]
```diff
@@ -1293,6 +1293,14 @@ class Comment : Item
     override @property bool isEmptyXML() const { return false; } /// Returns false always
 }
 
+unittest // issue 16241
+{
+    import std.exception: assertThrown;
```
Comment (burner) [inline std/xml.d]: exception: assert -> exception : assert

**Comment by burner** [comment]
I overlooked the colon style

**Comment by burner** [comment]
other then that still LGTM

**Comment by aG0aep6G** [comment]
> I overlooked the colon style

Fixed.

**Comment by quickfur** [comment]
LGTM

**Comment by quickfur** [comment]
Hmm. Why is the CI failing?

**Comment by quickfur** [comment]
Ah, I get it, we need to rebase on HEAD because PR #4569 is needed for CI to pass.

@aG0aep6G Please rebase, then this is good to go. Thanks!

**Comment by aG0aep6G** [comment]
I have amended the commit with nothing, still changing the hash and forcing a rebuild. I have not rebased. But Travis CI is green now. Looks like it doesn't re-run when the base changes. Not good.

**Comment by quickfur** [comment]
Auto-merge toggled on

**Pull request merged**